src/reducers/versions.spec.tsx

```diff
@@ -117,12 +119,81 @@ describe(__filename, () => {
 
       expect(state).toEqual({
         ...initialState,
+        expandedPaths: getParentFolders(version.file.selected_file),
         versionInfo: {
           [version.id]: createInternalVersion(version),
         },
       });
     });
 
+    it('prunes expandedPaths when loading version info', () => {
+      const path1 = 'path/1';
+      const path2 = 'path/2';
+      const expandedPaths = [path1, path2, ROOT_PATH];
+      const oldVersionId = nextUniqueId();
+      const oldVersion = createExternalVersionWithEntries(
```

Comment:
This test sets up a situation where the expanded paths of an old version are filtered out according to a new version. However, that filtering logic is already tested [here](https://github.com/mozilla/addons-code-manager/blob/4efe46a9711b8491a5d2c4faf56720a7c41814ff/src/reducers/versions.spec.tsx#L3259) since it's part of the implementation of `getPrunedExpandedPaths()`. The reducer will be relying on that logic but all this test needs to do is assert that reducing the `loadVersionInfo` action calls `getPrunedExpandedPaths()` with:

1. the `expandedPaths` that exist in state; and
2. the newly created internal version object.

That's it. `getPrunedExpandedPath()` guarantees that it will prune paths given the correct input.

With that in mind, see if you can simplify this test setup. Think about how you only need to assert the two points I outlined above and not any of the logic contained *within* `getPrunedExpandedPath()`.

If you get stuck, let me know.